internal/display/rsl.go

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package display
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/gittuf/gittuf/internal/repository"
+)
+
+// ShowRSL displays the RSL using the pager defined by PAGER
+func ShowRSL() error {
```

Comment:
IMO this can be more generic, it accepts something that needs to be printed, a destination, and maybe a bool to indicate if it should be paged.

Reply:
Note that this allows for better testing too, you can pass in some arbitrary tempfile instead of os.Stdout in the test.